grudge/op.py

```diff
@@ -159,84 +159,66 @@ def _gradient_kernel(actx, out_discr, in_discr, get_diff_mat, inv_jac_mat, vec,
 
 # {{{ common derivative "helpers"
 
-def _div_helper(dcoll, diff_func, *args):
-    if len(args) == 1:
-        vecs, = args
-        dd = dof_desc.DOFDesc("vol", dof_desc.DISCR_TAG_BASE)
-    elif len(args) == 2:
-        dd, vecs = args
-    else:
-        raise TypeError("invalid number of arguments")
-
+def _div_helper(ambient_dim, component_div, is_scalar, vecs):
```

Comment:
Since I figure you're about to use these from mirgecom (i.e. outside of grudge), why not:
- move these to `grudge.tools`,
- make them public and give them better names,
- document them?

Reply:
Refactored some more. Thoughts?